samcli/hook_packages/terraform/hooks/prepare/resource_linking.py

```diff
@@ -2,3 +2,41 @@
 Use Terraform plan to link resources together
 e.g. linking layers to functions
 """
+
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional
+
+
+@dataclass
+class Value:
+    constant_value: Any
+    references: List[str]
+
+
+@dataclass
+class Module:
```

Comment:
There was a comment on the classes names to make it clear that it is related to terraform. I am not really good in naming, so I left this comment to let you think about the names :)

Reply:
I thought about naming, but I feel like this is clear enough within this module (if anybody imports it, they could just rename the import). Happy to hear others' thoughts though

Reply:
I'm not completely opposed to prepending the classes with `TF` (eg `TFResource` removes potential ambiguity about whether its a Terraform or CFN resource), but I'm also okay with leaving it as is for the reason you mentioned